tests/unit/fixtures/gameStates/cardValidationErrors.js

```diff
@@ -0,0 +1,189 @@
+import { Card } from '../Card';
+import GamePhase from '../../../../utils/GamePhase.json';
+import MoveType from '../../../../utils/MoveType.json';
+
+export const validationErrors = {
```

Comment:
I think it would make sense to remove the `validationErrors` layer from this file and export each obj separately like
```
const duplicateCard = { // .. };
const invalidCard = {};
const missingCard = {};

export {
  duplicateCard,
  missingCard,
  invalidCard, // etc
}
```

That way consumers can just
```
import { duplicateCard, missingCard, invalidCard } from 'path-to-here'
```